common/src/main/java/com/nearsoft/fgaribay/mgmt/config/BrokerConfig.java

```diff
@@ -0,0 +1,76 @@
+package com.nearsoft.fgaribay.mgmt.config;
+
+import org.springframework.amqp.core.*;
```

Comment:
Thanks, actually the IDE was the one to format it that way. IntelliJ uses by default wildcard imports when more than 5 classes from the same package are imported.

Should I disable it and update my code so that it matches the preferred style?

https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports